test/WebSites/FormatterWebSite/Controllers/TryValidateModelController.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace FormatterWebSite
+{
+    public class TryValidateModelController : Controller
+    {
+        [HttpGet]
+        public IActionResult GetInvalidUser()
+        {
+            var user = new User()
+            {
+                Id = 0,
+                Name = "x"
+            };
+            if (!TryValidateModel(user))
+            {
+                return new BadRequestObjectResult(ModelState);
```

Comment:
nit: add code comments to `if` and fall-through `return`s to make expected case obvious